docs/management/updates.md

```diff
@@ -113,6 +113,23 @@ Also known as RPU, **this is the advised way to update your pool**. By just clic
 This powerful and fully automated mechanism requires some prerequisites: all your VMs disks must be on a one (or more) shared storage. Also, high-availability will be automatically disabled, as the XO load balancer plugin and backup jobs. Everything will be enabled back when it's done!
 :::
 
+:::warning
+RPU is disabled for pools with LINSTOR SRs. The reason is that after a reboot of a just updated host, it's possible that it can no longer communicate with other hosts through LINSTOR satellites. In fact LINSTOR expects that we always use satellites and controllers with the same version.
```

Comment:
I would say XOSTOR since it's our product name

Reply:
To talk about SRs? I agree, but for satellites/controllers it seems more coherent to me to talk about LINSTOR.